packages/stylelint-config-nimble/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nimblehq/stylelint-config-nimble",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
As the last time, I tried to release this version to test the publishing workflow, should we bump it to a higher version?

Reply:
Okay, let's bump to `1.0.2` 😂